lib/facter/easyrsa.rb

```diff
@@ -12,7 +12,7 @@
       binaryv3 = '/usr/share/easy-rsa/3/easyrsa'
     when %r{Ubuntu|Debian}
       binaryv3 = case operatingsystemrelease
-                 when %r{|11|12|18.04|20.04|22.04|24.04}
```

Comment:
I think this case statement doesn't make sense anymore? the first when matches all operating system versions?

Reply:
also aren't these legacy-facts?

Reply:
oh yes, that should also be fixed in line 7/8.